src/pip/_internal/commands/configuration.py

```diff
@@ -180,6 +184,16 @@ def unset_name(self, options, args):
 
         self._save_configuration()
 
+    def list_config_files(self, options, args):
```

Comment:
You can add the type annotation for this method. :)

Reply:
Actually no other functions have been type-annotated here. I have 3 other PR's adding type annotations to other files in `commands`, probably will create one for this file in near future.